deno.json

```diff
@@ -1,29 +1,27 @@
 {
   "tasks": {
-    "chel": "deno task build && deno run --import-map=./vendor/import_map.json --allow-net --allow-read=. --allow-write=. --no-remote build/main.js",
+    "chel": "deno task build && deno run --allow-net --allow-read=. --allow-write=. build/main.js",
     "lint": "standard ./src",
```

Comment:
See my 'review' comment about this specific line. Also, an advantage of using `npx` is that a specific version can be pinned.